site.cfg.example

```diff
@@ -79,14 +79,12 @@
 
 # Defaults
 # ========
-# The settings given here will apply to all other sections if not overridden.
+# The settings here will apply to all sections as general defaults
 # This is a good place to add general library and include directories like
 # /usr/local/{lib,include}
-#
-#[ALL]
+#[DEFAULT]
 #library_dirs = /usr/local/lib
```

Comment:
Change these to `/usr/local/lib64:/usr/local/lib:/usr/lib64:/usr/lib:/usr/lib/` and `/usr/local/include:/usr/include` so it's clearer how to specify multiple paths and that default search paths will get overridden...

Reply:
adding, and adding a comment about using `os.pathsep`